diff -up container-selinux-2.237.0/container.te.397 container-selinux-2.237.0/container.te
--- container-selinux-2.237.0/container.te.397	2025-09-19 06:40:30.384659594 +0200
+++ container-selinux-2.237.0/container.te	2025-09-19 06:41:40.003332145 +0200
@@ -1326,6 +1326,9 @@ allow container_domain init_t:unix_strea
 
 allow container_t proc_t:filesystem remount;
 
+# Allow containers to access shared runtime directories for OCI runtime optimizations
+allow container_t container_var_run_t:dir list_dir_perms;
+
 # Container kvm - Policy for running kata containers
 container_domain_template(container_kvm, container)
 typeattribute container_kvm_t container_net_domain, container_user_domain;
