diff -up container-selinux-2.163.0/container.te.orig container-selinux-2.163.0/container.te
--- container-selinux-2.163.0/container.te.orig	2021-06-16 16:14:04.107700701 +0200
+++ container-selinux-2.163.0/container.te	2021-06-16 16:14:29.756010679 +0200
@@ -454,7 +454,7 @@ modutils_domtrans_kmod(container_runtime
 systemd_status_all_unit_files(container_runtime_domain)
 systemd_start_systemd_services(container_runtime_domain)
 systemd_dbus_chat_logind(container_runtime_domain)
-systemd_chat_resolved(container_runtime_domain)
+#systemd_chat_resolved(container_runtime_domain)
 
 userdom_stream_connect(container_runtime_domain)
 userdom_search_user_home_content(container_runtime_domain)
